engine/loader.asm

```diff
@@ -41,6 +41,8 @@ Ldr_ProgressBarPtr      zmb     2
 Ldr_ProgressBarPage     zmb     1
 Ldr_CurProgressSec      zmb     2
 Ldr_CurProgressPct      zmb     1
+                        zmb     1
+ObjCodeVirtualPage      zmb     1
```

Comment:
Rather than making this a global variable and initializing it in Ldr_Load_Level, I would make it a variable that's local to the function Ldr_Load_SpriteGroup and initialize it to #VH_LVLOBJCODE1 at the beginning of that function

Reply:
I did look into this and found an issue: `Ldr_Load_Level` is responsible for iterating through all of the sprite groups (e.g. 1, 2, 3, 5, 8) and we may cross the 8K barrier in the middle (say 3).  Thus `Ldr_Load_SpriteGroup` has to know when to initialize `ObjCodeVirtualPage` and when not to.

We can add a parameter to `Ldr_Load_SpriteGroup` to indicate that it is loading the first sprite group. Any thoughts here?